File: java/src/main/java/com/teamlab/kotlin/dagger/issue/di/ActivityComponent.java
package com.teamlab.kotlin.dagger.issue.di;

import com.teamlab.kotlin.dagger.issue.MainFragment;

import dagger.Component;

@Component(modules = ActivityModule.class, dependencies = ApplicationComponent.class)
@ActivityScope
public interface ActivityComponent {
    void inject(MainFragment fragment);
}

File: kotlin/src/main/kotlin/com/teamlab/kotlin/dagger/issue/di/ActivityComponent.kt
package com.teamlab.kotlin.dagger.issue.di

import com.teamlab.kotlin.dagger.issue.MainFragment

import dagger.Component

@Component(modules = arrayOf(ActivityModule::class), dependencies = arrayOf(ApplicationComponent::class))
@ActivityScope
interface ActivityComponent {
    fun inject(fragment: MainFragment)
}
